File: backend.Domain/Entities/Surah.cs
namespace backend.Domain.Entities;

public class Surah
{
    public int Id { get; set; }
    public int SurahNumber { get; set; }
    public string NameArabic { get; set; } = string.Empty;
    public string NameEnglish { get; set; } = string.Empty;
    public string NameBangla { get; set; } = string.Empty;
    public int TotalAyahs { get; set; }
}

File: android/app/src/main/java/com/example/quran_app/domain/model/Surah.kt
package com.example.quran_app.domain.model

import com.google.gson.annotations.SerializedName

data class Surah(
    @SerializedName("id") val id: Int,
    @SerializedName("surahNumber") val surahNumber: Int,
    @SerializedName("nameArabic") val nameArabic: String,
    @SerializedName("nameEnglish") val nameEnglish: String,
    @SerializedName("nameBangla") val nameBangla: String,
    @SerializedName("totalAyahs") val totalAyahs: Int
)
